### ISSUE TITLE
Unable to upload from AppVeyor

### ISSUE THREAD

**Issue by altendky** [codecov/codecov-python#170]
v2.0.15: https://ci.appveyor.com/project/sunu/qt5reactor/builds/20136162/job/oy4qq48g746thmrx

Specifically note `XX> Searching for reports disabled` (#138) and `Error: 404 Client Error: Not Found for url:` (perhaps a new message related to #123).  The net result is that I don't get coverage uploaded from AppVeyor for these builds.

v2.0.5: https://ci.appveyor.com/project/sunu/qt5reactor/builds/20136639/job/b3ui3t6h9i14ovkt

The search being disabled can still be avoided by dropping back to v2.0.5 before https://github.com/codecov/codecov-python/commit/d334f5c53def608a502f520c3856e115c34f5086 but the 404 still remains.

```
tox -e codecov
GLOB sdist-make: C:\projects\qt5reactor\setup.py
codecov create: C:\projects\qt5reactor\.tox\codecov
codecov installdeps: codecov==2.0.15
codecov installed: certifi==2018.10.15,chardet==3.0.4,codecov==2.0.15,coverage==4.5.1,idna==2.7,requests==2.20.0,urllib3==1.24.1
codecov runtests: PYTHONHASHSEED='190'
codecov runtests: commands[0] | python -c 'import sys; print(sys.version)'
3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
codecov runtests: commands[1] | codecov
The filename, directory name, or volume label syntax is incorrect.
FIND: Parameter format not correct
      _____          _
     / ____|        | |
    | |     ___   __| | ___  ___ _____   __
    | |    / _ \ / _  |/ _ \/ __/ _ \ \ / /
    | |___| (_) | (_| |  __/ (_| (_) \ V /
     \_____\___/ \____|\___|\___\___/ \_/
                                    v2.0.15
==> Detecting CI provider
    AppVeyor Detected
    Fixing merge commit SHA
==> Preparing upload
    Error running `cd 'C:\projects\qt5reactor' && git ls-files`: None
==> Processing gcov (disable by -X gcov)
    Executing gcov (find 'C:\projects\qt5reactor' -not -path './bower_components/**' -not -path './node_modules/**' -not -path './vendor/**' -type f -name '*.gcno'  -exec gcov -pb  {} +)
    Error running `find 'C:\projects\qt5reactor' -not -path './bower_components/**' -not -path './node_modules/**' -not -path './vendor/**' -type f -name '*.gcno'  -exec gcov -pb  {} +`: None
==> Collecting reports
XX> Searching for reports disabled
    Generating coverage xml reports for Python
    + C:\projects\qt5reactor\coverage.xml bytes=11471
==> Appending environment variables
    + TOXENV
==> Uploading
    .url https://codecov.io
    .query commit=933e8c1eb35d9cdf5be185d9aaa64986fd38b607&branch=master&job=sunu%2Fqt5reactor%2Fv0.5.post75%2Bge9854d7%20b49&pr=24&service=appveyor&build=oy4qq48g746thmrx&slug=sunu%2Fqt5reactor&package=py2.0.15
    Pinging Codecov...
    Direct to s3 failed. Using backup v2 endpoint.
    Uploading to Codecov...
    HTTP 404
Not Found
Error: 404 Client Error: Not Found for url: https://codecov.io/upload/v2?commit=933e8c1eb35d9cdf5be185d9aaa64986fd38b607&branch=master&job=sunu%2Fqt5reactor%2Fv0.5.post75%2Bge9854d7%20b49&pr=24&service=appveyor&build=oy4qq48g746thmrx&slug=sunu%2Fqt5reactor&package=py2.0.15
Tip: See all example repositories: https://github.com/codecov?query=example
Support channels:
  Email:   hello@codecov.io
  IRC:     #codecov
  Gitter:  https://gitter.im/codecov/support
  Twitter: @codecov
___________________________________ summary ___________________________________
  codecov: commands succeeded
  congratulations :)
```

**Comment by raczben** [comment]
Please read https://github.com/codecov/codecov-python/issues/158#issuecomment-514282362

Does your problem permanent or occasional?

**Comment by altendky** [comment]
This is really old but as I recall it was an every time sort of thing.  I traced the first point down and found the line of code and introducing commit that explicitly disabled collection on AppVeyor and documented it in codecov/support#402 but at some point codecov/support got removed.  I don't recall the second 404 issue particularly.

Since this issue references trouble with 2.0.15 and that is still the latest on PyPI and the collection issue looked to be the direct result of a fairly direct line of code...  I guess this really still ought to be open.

Maybe https://github.com/codecov/codecov-python/commit/d334f5c53def608a502f520c3856e115c34f5086?  And yes, it is still there.

https://github.com/codecov/codecov-python/blob/73b1b138ec40912eee6fed02636be6011105706e/codecov/__init__.py#L428

For completeness, here's the code that checks that and writes out the above mentioned message.

https://github.com/codecov/codecov-python/blob/73b1b138ec40912eee6fed02636be6011105706e/codecov/__init__.py#L640-L641

@stevepeak, it was a long time ago but any recollection on this?  Is searching really supposed to be explicitly and unconditionally disabled on AppVeyor?  Or maybe I'm misunderstanding the importance of 'searching'?
